flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -188,6 +188,26 @@ class AndroidFlutterLocalNotificationsPlugin
   Future<bool?> requestNotificationsPermission() async =>
       _channel.invokeMethod<bool>('requestNotificationsPermission');
 
+  /// Requests access to notification policy. Returns whether the
```

Comment:
Can you separate the sentences? See https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph